lib/reporter.js

```diff
@@ -115,17 +116,35 @@ class AllureReporter extends events.EventEmitter {
     }
 
     runnerCommand (command) {
-        const allure = this.getAllure(command.cid)
+        if (!this.options.disableWebdriverStepsReporting) {
```

Comment:
How about
```js
if (this.options.disableWebdriverStepsReporting) {
    return
}
```
so instead of wrapping the whole thing into an if, let's return early and save the indentation